code/game/gamemodes/game_mode.dm

```diff
@@ -377,6 +378,7 @@
 /datum/game_mode/proc/get_players_for_role(role, pre_do, check_pq = TRUE)
 	var/list/players = list()
 	var/list/candidates = list()
+//	var/pass = TRUE
```

Comment:
I imagine this slipped in accidentally

Reply:
This was during the Seelie bandit fixes, originally Fenix tried a true / false flag that would get flipped the second that one Seelie became bandit. It ended up not working because all Seelie that sign up for bandit get it at once I believe, making the pass flag get flipped too late to matter. It can be removed